assets/scss/biz-portal.scss

```diff
@@ -22,6 +22,10 @@ body {
   overflow: auto;
   position: relative;
 }
+.fa-list-icon {
+  margin-right: 4px;
```

Comment:
This used to be 8px with the old icons.

- please remove the old now dead css
- please make this 9px to match the mockup

![Screenshot_2019-07-29_18-26-05](https://user-images.githubusercontent.com/235801/62065239-ddbe8080-b22e-11e9-8084-aa660668e6fa.png)

Reply:
I've removed the old CSS and moved the new icon class into the same two subclasses as the old css was, and I've updated the margin to 9px.